.gitignore

```diff
@@ -1,3 +1,5 @@
+.DS_Store
```

Comment:
This should be in your own global `.gitignore`. Only project-specific ignores should be in the repo.